### PR TITLE
making date-range testing helper functions

### PR DESCRIPTION
**Pull request by sbillinge**
If I can give ``since`` and ``before`` date ranges on the command line, I would like to have easy to use helper functions that can test if something is valid or not in date range so I added some functions to tools.py.  Similar to ``is_current`` etc. in cpbuilder, but just tests two dates, but takes them in the format that they come from the db for convenience.

### CODE REVIEW

File: regolith/tools.py [hunk 1]
```diff
@@ -116,6 +117,28 @@ def month_and_year(m=None, y=None):
     return "{0} {1}".format(SHORT_MONTH_NAMES[m], y)
 
 
+def is_since(y, sy, m=1, d=1, sm=1, sd=1):
```
Comment (scopatz) [inline regolith/tools.py]: Can you please add docstrings for these functions and explain what the meaning of the function arguments is here? Thanks!

**Comment by scopatz** [comment]
Thankd for putting this in @sbillinge! Just some docstrings needed

**Comment by sbillinge** [comment]
yes, of course.  
I am torn and perhaps @scopatz @CJ-Wright  you can weigh in.  

Actually I started the docstrings but my pycharm helper was making rst format and I think we want numpy style, and I couldn't figure out how to switch that without googlin and I was on the plane.  Sorry, tmi, but it reminds me I need to google that....

**Comment by CJ-Wright** [comment]
https://www.jetbrains.com/help/pycharm/settings-tools-python-integrated-tools.html

**Comment by sbillinge** [comment]
@scopatz I think this is ready now, though I see one push I accidentally made on the wrong branch.....but it affects a different file.

File: news/date_range_filter.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+**Added:**
+ * filters in tools.py that return true if a given date is since or before or
```
Comment (CJ-Wright) [inline news/date_range_filter.rst]:
```suggestion
 * filters in ``regolith.tools`` that return true if a given date is since or before or
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**